<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Urania</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>596</width>
      <height>341</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="hsomsTab">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <attribute name="title">
      <string>HSOMs</string>
     </attribute>
     <widget class="QTableWidget" name="musesTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>571</width>
        <height>261</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <column>
       <property name="text">
        <string>File Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SOM Width</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SOM Height</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Feature Width</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Feature Height</string>
       </property>
      </column>
     </widget>
     <widget class="QToolButton" name="addBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QToolButton" name="remBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>280</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QToolButton" name="newBtn">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>280</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="testTab">
     <attribute name="title">
      <string>Test</string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Training Directory:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="testDirBrowseBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>25</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="testDirTextBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>/data/suspect_library/small_test</string>
      </property>
     </widget>
     <widget class="QPushButton" name="testBtn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>285</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Test!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>46</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Direct Hit %:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="testHitTextBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="testMissTextBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>155</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Wide Miss %:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="testNearTextBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>125</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Near Miss %:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="testDetailsTextBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>456</width>
        <height>266</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Monospace&quot;;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="classifyTab">
     <attribute name="title">
      <string>Classify</string>
     </attribute>
     <widget class="QLineEdit" name="classifyDirTextBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>/data/suspect_library/test_sets/SetA</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Target Directory:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="classifyDirBrowseBtn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>10</y>
        <width>25</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="classifyThumbGraphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>291</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="classifyVoteTextBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>291</width>
        <height>111</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTreeView" name="classifyDirTreeView">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>271</width>
        <height>261</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="iconSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>595</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionNew_HSOM"/>
    <addaction name="actionRemove_HSOM"/>
   </widget>
   <widget class="QMenu" name="menuDemos">
    <property name="title">
     <string>demos</string>
    </property>
    <addaction name="actionRect_tester"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDemos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_As">
   <property name="text">
    <string>Save HSOM As</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save HSOM</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open HSOM</string>
   </property>
  </action>
  <action name="actionNew_HSOM">
   <property name="text">
    <string>New HSOM</string>
   </property>
  </action>
  <action name="actionRemove_HSOM">
   <property name="text">
    <string>Remove HSOM</string>
   </property>
  </action>
  <action name="actionRect_tester">
   <property name="text">
    <string>rect tester</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>testDirTextBox</tabstop>
  <tabstop>testDirBrowseBtn</tabstop>
  <tabstop>testBtn</tabstop>
  <tabstop>testDetailsTextBox</tabstop>
  <tabstop>testHitTextBox</tabstop>
  <tabstop>testNearTextBox</tabstop>
  <tabstop>testMissTextBox</tabstop>
  <tabstop>classifyDirTextBox</tabstop>
  <tabstop>classifyDirBrowseBtn</tabstop>
  <tabstop>classifyThumbGraphicsView</tabstop>
  <tabstop>classifyVoteTextBox</tabstop>
  <tabstop>classifyDirTreeView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
